pkg/storage/store.go

```diff
@@ -10,9 +11,10 @@ import (
 
 // Store performs database operations for EASi
 type Store struct {
-	DB     *sqlx.DB // temporarily export until SystemIntakesHandler doesn't take db
-	logger *zap.Logger
-	clock  clock.Clock
+	DB        *sqlx.DB // temporarily export until SystemIntakesHandler doesn't take db
```

Comment:
This no longer needs to be exported, per the comment. Can you make that field unexported?